platform/linux-generic/pktio/dpdk.c

```diff
@@ -163,6 +162,11 @@ static struct rte_mempool *mbuf_pool_create(const char *name,
 	unsigned num;
 	uint16_t data_room_size;
 
+	if (!(pool_entry->mem_from_huge_pages)) {
+		ODP_ERR("DPDK requires memory is allocated from huge pages\n");
+		return NULL;
```

Comment:
Is this true? This code was passing prior to this test being added, so was that just luck? I can understand that huge pages are preferred, but are they required?